lib/module.js

```diff
@@ -265,7 +265,7 @@ if (process.platform === 'win32') {
       // The path segment separator check ('\' and '/') was used to get
       // node_modules path for every path segment.
       // Use colon as an extra condition since we can get node_modules
-      // path for dirver root like 'C:\node_modules' and don't need to
+      // path for drive root like 'C:\node_modules' and don't need to
       // parse driver name.
```

Comment:
I went back and forth a few times on the meaning on the sentence. In the end I thought best to just correct the spelling mistake incase I was wrong. But I can change it to `parse drive name` if that is the consensus.